src/util/DataUtils.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable import/prefer-default-export */
+/**
+ * @module DataUtils
+ */
+
+/**
+ * Dummy comment needed for correct doc display, do not remove.
+ */
+import { checkAddress } from '@polkadot/util-crypto'
+import IAttestedClaim from '../types/AttestedClaim'
+import { NonceHash } from '../types/RequestForAttestation'
+import Identity from '../identity/Identity'
+import AttestedClaim from '../attestedclaim/AttestedClaim'
+import { hashObjectAsStr, verify } from '../crypto/Crypto'
+
+export function validateAddress(
```

Comment:
it's tempting to introduce an `Address` type here and make this the type guard. Do you think that's overkill? Same for `validateHash` below.

Reply:
Yes it's tempting to type everything, but type information just does not exist post compilation.
As in something with type address would be actually of type string at runtime.

Implementing it as types would make no difference at runtime, although maybe providing "some" usefulness at pre compilation level?

Reply:
This would only help pre-compilation to signal that we have checked some assumption about that string at this point, which may (or may not) be important at a later point.

Reply:
@LeonFLK please make a ticket for that. doesn't have to be done in this PR